Sources/SwiftKuery/Table.swift

```diff
@@ -19,7 +19,7 @@
 /// Subclasses of the Table class are metadata describing a table in a relational database that you want to work with.
 open class Table: Buildable {
     var _name = ""
-    private var columns: [Column]
+    public private (set) var columns: [Column]
```

Comment:
A couple of points:

  1) What do you do when the developer hasn't specified schema information?
  2) Why isn't that in the Swift-Kuery "core", with The connection adding formaters to the QueryBuilder "parameterization"?